bot.py

```diff
@@ -106,6 +107,10 @@ async def on_message(self, message):
         
     string_channel_id = str(message.channel.id)
     
+    if self.chat_channel_id:
```

Comment:
Would be nice with a very short comment, that explains this

Reply:
If `self.chat_channel_id` is not empty.

Reply:
Done.